File: System/Common/Temperature.cs
using System;
using System.Dynamic;

namespace Common
{
    public class Temperature : Measurement
    {
        public long PatientID { get; set; }

        public double temperature { get; set; }

        public Temperature(long PatientID, string timestamp, double temperature)
        {
            this.PatientID = PatientID;
            this.Timestamp = timestamp;
            this.temperature = temperature;
        }

        public Boolean isCritical()
        {
            if (temperature < 36.5 || temperature > 37.5)
                return true;
            else
                return false;
        }

    }
}

File: System/Model/src/Model/Temperature.java
package Model;

public class Temperature extends Measurement {
    private double temperature;

    public Temperature(long patientId, String timestamp, double temperature) {
        super(patientId, timestamp);
        this.temperature = temperature;
    }


    public double getTemperature() {
        return temperature;
    }

    public boolean isCritical() {
        return temperature < 36.5 || temperature > 37.5;
    }

    public String toString() {
        return super.getTimestamp() + "	" + temperature;
    }
}
